test/SlowTests/Issues/FilteredReplicationTests.cs

```diff
@@ -98,6 +98,7 @@ public async Task Can_Setup_Filtered_Replication()
             await storeA.Maintenance.SendAsync(new PutPullReplicationAsHubOperation(new PullReplicationDefinition
             {
                 Name = "pull",
+                WithFiltering = true
```

Comment:
do we need explicit field for that? Can we infer that from configuration ?

Reply:
as discussed, leaving this